jupyter_core/command.py

```diff
@@ -57,8 +58,8 @@ def list_subcommands():
         except OSError:
             continue
         for name in names:
-            if name.startswith('jupyter-'):
-                subcommands.add(name.split('-')[1])
```

Comment:
This is deliberate, so that it only goes one level down the hierarchy. It shouldn't list every subcommand of every subcommand as a subcommand of the highest level.